src/applications/personalization/dashboard/helpers.jsx

```diff
@@ -28,6 +28,7 @@ import feedbackConfig from 'applications/edu-benefits/feedback-tool/config/form.
 import burialsConfig from 'applications/burials/config/form.js';
 import edu1990Config from 'applications/edu-benefits/1990/config/form.js';
 import edu1995Config from 'applications/edu-benefits/1995/config/form.js';
+import edu1995StemConfig from 'applications/edu-benefits/1995-STEM/config/form.js';
```

Comment:
this should have been part of https://github.com/department-of-veterans-affairs/vets-website/pull/10043